index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title></title>
+        <link rel="icon" href="" type="image/gif" sizes="">
+        <link href="css/reset.css" type="text/css" rel="stylesheet">
+        <link href="css/style.css" type="text/css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300&display=swap" rel="stylesheet">
+
+    </head>
+    <body>
+        <header>
+            <section id="nav">
```

Comment:
<nav> is intended to be used to wrap the entire navigation menu, so instead of using a <section> with an id of 'nav', this should perhaps just be a <nav>. You always want to put the links inside of a list, this is important for accessibility as people who use screen-readers will know how many links there are (the screen-reader will say something like, "Home, item 1 of 4")